apps/backend/src/api/devices.py

```
@@ -8,41 +8,45 @@
 
-from fastapi import APIRouter, Depends, HTTPException, Query, Path
+from fastapi import APIRouter, Depends, HTTPException, Path, Query
 from sqlalchemy.ext.asyncio import AsyncSession
 
+from apps.backend.src.api.common import get_current_user
 from apps.backend.src.core.database import get_db_session
-from apps.backend.src.utils.ssh_config_parser import parse_ssh_config
-from apps.backend.src.schemas.device import DeviceImportResult
 from apps.backend.src.core.exceptions import (
-    DeviceNotFoundError,
     DatabaseOperationError,
```

Ruff (0.12.2)

`apps.backend.src.core.exceptions.DatabaseOperationError` imported but unused

Remove unused import: `apps.backend.src.core.exceptions.DatabaseOperationError`

(F401)